src/app/files.js

```diff
@@ -107,6 +107,47 @@ function Files (storage) {
     return files
   }
 
+  //
+  // Tree model for files
+  // {
+  //   'a': { }, // empty directory 'a'
+  //   'b': {
+  //     'c': {}, // empty directory 'b/c'
+  //     'd': { '/readonly': true, '/content': 'Hello World' } // files 'b/c/d'
+  //     'e': { '/readonly': false, '/path': 'b/c/d' } // symlink to 'b/c/d'
```

Comment:
Actually i start to like the way you do stuff :-)
Maybe it could be added ad `/props` the way you do it now, so the initial file tree from above would look like:

```js
var fs = {
  "a" : { // empty directory
    "/type": "directory",
    "/dev": 2114,
    "/ino": 48064969,
    "/mode": 33188,
    "/nlink": 1,
    "/uid": 85,
    "/gid": 100,
    "/rdev": 0,
    "/size": 527,
    "/blksize": 4096,
    "/blocks": 8,
    "/atime": "Mon, 10 Oct 2011 23:24:11 GMT",
    "/mtime": "Mon, 10 Oct 2011 23:24:11 GMT",
    "/ctime": "Mon, 10 Oct 2011 23:24:11 GMT",
    "/birthtime": "Mon, 10 Oct 2011 23:24:11 GMT"
  },
  "b": {
    "/type": "directory",
    // ... and more meta as above ...
    "c": {
      "/type": "directory",
      // ... and more meta as above ...
    },
    "d": {
      "/type": "file",
      // ... and more meta as above ...
      "": "Hello World"
    },
    "e": {
      "/type": "symlink",
      // ... and more as above ...
      "": ["b"]
    },
    "f": {
      "/type": "file",
      // ... and more meta as above ...
      "": "...maybe javascript content or base64 encoded stuff or whatever ..."
      // "": function () { console.log('executable stuff') }
    }
  }
}

```